be/src/exec/olap_common.h

```diff
@@ -386,8 +391,8 @@ class ColumnValueRange {
 
     static ColumnValueRange<primitive_type> create_empty_column_value_range(
             const std::string& col_name, int precision, int scale) {
-        return ColumnValueRange<primitive_type>(col_name, TYPE_MAX, TYPE_MIN, false, precision,
-                                                scale);
+        return ColumnValueRange<primitive_type>(col_name, TYPE_MAX, TYPE_MIN, true, false,
```

Comment:
the function should be check is_nullable should be `true` ?

Reply:
is_nullable_col has been passed in as a parameter